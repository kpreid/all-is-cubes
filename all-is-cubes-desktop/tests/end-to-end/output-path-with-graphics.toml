# Tests what happens if `-o` *and* a graphics option are set.
# The current answer is: produce both, but the adjustments for recording also influence what is printed.
# Not ideal but that's more of a problem with recording in general.
bin.name = "all-is-cubes"
args = "--no-config-files --simplify-log-format -Slighting_display='\"None\"' --template=lighting-bench --seed=0 --graphics=print --output=test-output-path-with-graphics.png --display-size 40x40"
status.code = 0
