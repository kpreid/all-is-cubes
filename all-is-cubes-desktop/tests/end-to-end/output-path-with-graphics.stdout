[?25l[0m[38;5;0m[49m                            [38;5;145m[49mâ–„[38;5;235m[49mâ–„[38;5;246m[49mâ–„[38;5;236m[49mâ–„[38;5;246m[49mâ–„â–„[38;5;145m[49mâ–„[38;5;235m[49mâ–„â–„[38;5;236m[49mâ–„[38;5;246m[49mâ–„â–„[0m
[38;5;0m[49m                            [38;5;145m[48;5;138mâ–„[38;5;145m[48;5;235mâ–„[38;5;236m[48;5;246mâ–„[39m[48;5;246mâ–„[38;5;247m[48;5;246mâ–„[39m[48;5;245mâ–„[38;5;236m[48;5;145mâ–„[38;5;145m[48;5;235mâ–„â–„[39m[48;5;235mâ–„[38;5;145m[48;5;245mâ–„â–„[0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m   [38;5;231m[49mâ–„[38;5;0m[48;5;231m  [38;5;171m[48;5;231mâ–„[38;5;171m[48;5;177mâ–„â–„â–„[38;5;177m[48;5;231mâ–„[38;5;0m[48;5;231m       [38;5;0m[48;5;108m    [38;5;0m[48;5;231m  [38;5;0m[48;5;62m    [38;5;0m[48;5;231m  [38;5;0m[48;5;181m [38;5;181m[48;5;231mâ–„[38;5;0m[48;5;217m [38;5;181m[48;5;231mâ–„[38;5;0m[48;5;231m  [38;5;231m[49mâ–„[38;5;0m[49m   [0m
[38;5;231m[49mâ–„[38;5;0m[48;5;231m   [38;5;249m[48;5;231mâ–„[38;5;0m[48;5;249m [38;5;249m[48;5;171mâ–„â–„â–„â–„â–„[38;5;0m[48;5;192m     [38;5;0m[48;5;249m  [38;5;176m[48;5;212mâ–„[38;5;212m[48;5;176mâ–„[38;5;212m[48;5;108mâ–„[38;5;176m[48;5;212mâ–„[38;5;0m[48;5;249m  [38;5;249m[48;5;62mâ–„â–„â–„â–„[38;5;0m[48;5;249m [38;5;249m[48;5;174mâ–„[38;5;249m[48;5;16mâ–„[38;5;0m[48;5;146m  [38;5;16m[48;5;146mâ–„[38;5;0m[48;5;146m [38;5;249m[48;5;231mâ–„[38;5;0m[48;5;231m   [38;5;231m[49mâ–„[0m
[38;5;0m[48;5;231m [38;5;167m[48;5;231mâ–„[38;5;0m[48;5;167m  [38;5;167m[48;5;249mâ–„[38;5;0m[48;5;167m  [38;5;249m[48;5;131mâ–„â–„[38;5;0m[48;5;249m [38;5;0m[48;5;176m  [38;5;249m[48;5;176mâ–„[38;5;0m[48;5;176m [38;5;249m[48;5;192mâ–„[38;5;249m[48;5;247mâ–„[38;5;0m[48;5;249m  [38;5;146m[48;5;238mâ–„[38;5;249m[48;5;97mâ–„[38;5;146m[48;5;176mâ–„[38;5;146m[48;5;238mâ–„[38;5;0m[48;5;249m   [38;5;0m[48;5;62m     [38;5;62m[48;5;249mâ–„[38;5;249m[48;5;146mâ–„â–„â–„â–„â–„[38;5;0m[48;5;249m [38;5;249m[48;5;231mâ–„â–„[38;5;0m[48;5;231m [0m
[38;5;0m[48;5;194m     [38;5;0m[48;5;167m [38;5;167m[48;5;16mâ–„[38;5;217m[48;5;167mâ–„[38;5;0m[48;5;217m      [38;5;0m[48;5;176m [38;5;0m[48;5;249m   [38;5;0m[48;5;146m  [38;5;146m[48;5;248mâ–„[38;5;248m[48;5;146mâ–„[38;5;0m[48;5;249m   [38;5;0m[48;5;62m      [38;5;0m[48;5;249m [38;5;249m[48;5;55mâ–„[38;5;0m[48;5;55m  [38;5;0m[48;5;171m     [0m
[38;5;0m[48;5;194m     [38;5;0m[48;5;249m  [38;5;0m[48;5;217m       [38;5;0m[48;5;249m                     [38;5;0m[48;5;171m     [0m
[38;5;0m[48;5;194m    [38;5;249m[48;5;188mâ–„[38;5;0m[48;5;249m  [38;5;186m[48;5;217mâ–„[38;5;0m[48;5;217m [38;5;187m[48;5;217mâ–„[38;5;186m[48;5;217mâ–„[38;5;0m[48;5;217m  [38;5;248m[48;5;217mâ–„[38;5;0m[48;5;249m   [38;5;0m[48;5;169m   [38;5;153m[48;5;169mâ–„[38;5;0m[48;5;169m [38;5;153m[48;5;169mâ–„[38;5;0m[48;5;249m   [38;5;249m[48;5;164mâ–„[38;5;0m[48;5;164m [38;5;188m[48;5;164mâ–„â–„â–„â–„[38;5;188m[48;5;249mâ–„â–„â–„[38;5;249m[48;5;170mâ–„[38;5;0m[48;5;171m   [38;5;116m[48;5;171mâ–„[0m
[38;5;0m[48;5;249m    [38;5;0m[48;5;151m [38;5;0m[48;5;186m [38;5;191m[48;5;150mâ–„[38;5;186m[48;5;145mâ–„[38;5;186m[48;5;150mâ–„[38;5;186m[48;5;151mâ–„[38;5;145m[48;5;150mâ–„[38;5;150m[48;5;186mâ–„[38;5;0m[48;5;249m    [38;5;153m[48;5;249mâ–„[38;5;0m[48;5;153m    [38;5;147m[48;5;153mâ–„[38;5;249m[48;5;153mâ–„â–„[38;5;0m[48;5;249m    [38;5;0m[48;5;188m        [38;5;188m[48;5;249mâ–„[38;5;0m[48;5;249m  [38;5;0m[48;5;116m [0m
[38;5;157m[48;5;248mâ–„[38;5;249m[48;5;248mâ–„[38;5;0m[48;5;249m  [38;5;145m[48;5;150mâ–„[38;5;0m[48;5;186m  [38;5;186m[48;5;150mâ–„[38;5;145m[48;5;150mâ–„[38;5;150m[48;5;186mâ–„[38;5;186m[48;5;145mâ–„[38;5;186m[48;5;249mâ–„[38;5;0m[48;5;249m    [38;5;0m[48;5;153m  [38;5;147m[48;5;153mâ–„[38;5;0m[48;5;153m  [38;5;66m[48;5;240mâ–„[38;5;0m[48;5;153m  [38;5;0m[48;5;249m   [38;5;249m[48;5;248mâ–„[38;5;0m[48;5;188m         [38;5;0m[48;5;249m [38;5;249m[48;5;115mâ–„[38;5;115m[48;5;116mâ–„[0m
[38;5;0m[48;5;249m   [38;5;249m[48;5;151mâ–„[38;5;150m[48;5;186mâ–„[38;5;151m[48;5;150mâ–„[38;5;0m[48;5;150m [38;5;0m[48;5;186m [38;5;0m[48;5;150m [38;5;145m[48;5;151mâ–„[38;5;150m[48;5;249mâ–„[38;5;249m[48;5;151mâ–„[38;5;0m[48;5;249m    [38;5;16m[48;5;153mâ–„â–„[38;5;0m[48;5;153m [38;5;16m[48;5;153mâ–„â–„[38;5;0m[48;5;153m  [38;5;249m[48;5;16mâ–„[38;5;0m[48;5;249m    [38;5;247m[48;5;188mâ–„[38;5;0m[48;5;188m        [38;5;0m[48;5;249m   [0m
[38;5;0m[48;5;249m                     [38;5;249m[48;5;247mâ–„[38;5;249m[48;5;248mâ–„[38;5;0m[48;5;249m     [38;5;249m[48;5;248mâ–„[38;5;249m[48;5;247mâ–„â–„â–„â–„â–„â–„â–„[38;5;249m[48;5;248mâ–„[38;5;0m[48;5;249m   [0m
[38;5;0m[48;5;249m                                        [0m
[38;5;0m[48;5;248m                                        [0m
[38;5;0m[48;5;248m                                        [0m
[38;5;0m[48;5;248m                                        [0m
[38;5;0m[48;5;248m                                        [0m
[38;5;0m[48;5;248m                                        [0m
[39m[48;5;248mâ–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„â–„[0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m                                        [0m
[38;5;0m[49m   [38;5;253m[49mâ–„[38;5;0m[49m      [38;5;253m[49mâ–„[38;5;0m[49m     [38;5;253m[49mâ–„[38;5;0m[49m      [38;5;253m[49mâ–„[38;5;0m[49m     [38;5;253m[49mâ–„[38;5;0m[49m      [38;5;253m[49mâ–„[38;5;0m[49m   [0m
[38;5;0m[49m [39m[48;5;251mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;253mâ–„[38;5;0m[49m   [39m[48;5;253mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;252mâ–„[38;5;0m[49m   [39m[48;5;253mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;251mâ–„[38;5;0m[49m  [39m[48;5;251mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;253mâ–„[38;5;0m[49m   [39m[48;5;252mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;253mâ–„[38;5;0m[49m   [39m[48;5;253mâ–„[39m[48;5;243mâ–„â–„[39m[48;5;251mâ–„[38;5;0m[49m [0m
[0m[0m[39m[49m[?25h[?1006l[?1015l[?1003l[?1002l[?1000l